src/test/protocol/STTx_test.cpp

```diff
@@ -1419,16 +1419,16 @@ class STTx_test : public beast::unit_test::suite
         try
         {
             protocol::TMTransaction tx2;
-            tx2.ParseFromArray(payload1, sizeof(payload1));
+            BEAST_EXPECT(! tx2.ParseFromArray(payload1, sizeof(payload1)));
```

Comment:
Might be nice to add a comment along the line that, "Depending on the version of protobuf either an exception is thrown of a zero-length transaction is returned.  In either case an exception is thrown, either here or by the `STTx` constructor below."  Or if that's not accurate, then whatever the truth is...

Reply:
👍 - let me know if the added comments make sense.

Reply:
Excellent.  Nicely done. Thanks.  :+1: